modules/cluster-logging-dashboards-es.adoc

```diff
@@ -0,0 +1,196 @@
+* 2 - Indicates that the Elasticsearch instance is in red status, which means that at least one primary shard and its replicas are not allocated.
+
+|Cluster nodes 
+|The total number of Elasticsearch nodes in the cluster.
+
+|Cluster data nodes 
+|The number of Elasticsearch data nodes in the cluster.
+
+|Cluster pending tasks 
+|The number of cluster state changes that have not been executed and are waiting in a cluster queue. For example: index creation, index deletion, or shard allocation. A growing trend indicates that the cluster is not able to keep up with changes.
+
+|===
+
+Elasticsearch cluster index shard status::
+
+Each Elasticsearch index is a logical group of one or more shards, which are basic units of persisted data. There are two types of index shards: primary shards, and replica shards. When a document is indexed into an index, it is stored in one of its primary shards and copied into every replica of that shard. The number of primary shards is specified when the index is created and the number cannot change during index lifetime. The number of replica shards can be changed at any time.
+
+The index shard can be in several states depending on its lifecycle phase or events occurring in the cluster. When the shard is able to perform search and indexing requests, the shard is active. If the shard cannot perform these requests, the shard is non–active. A shard might be non-active if the shard is initializing, reallocating, unassigned, and so forth.
+
+Index shards consist of a number of smaller internal blocks, called index segments, which are physical representations of the data. An index segment is a relatively small, immutable Lucene index that is created when Lucene commits newly-indexed data. Lucene merges index segments into larger segments in the background to keep the total number of segments low. If the process of merging segments is slower than the speed at which new segments are created, it could indicate a problem.
+
+When Lucene performs data operations, such as a search operation, Lucene performs the operation against the index segments in the relevant index. For that purpose, each segment contains specific data structures that are loaded in the memory and mapped. Index mapping can have a significant impact on the memory used by segment data structures.
+
+The *Logging/Elasticsearch Nodes* dashboard contains the following charts about the Elasticsearch index shards.
+
+.Elasticsearch cluster shard status charts
+[options="header"]
+
+|===
+|Metric|Description
+
+|Cluster active shards 
+|The number of active primary shards and the total number of shards, including replicas, in the cluster. If the number of shards grows higher, the cluster performance can start degrading.
+
+|Cluster initializing shards 
+|The number of non-active shards in the cluster. A cluster typically has non–active shards for short periods. A growing number of non–active shards over longer periods could indicate a problem.
```

Comment:
If you have another sentence about why a cluster only has non-active shards for a short period or the problem, it might be useful to add.